lib/phoenix/endpoint.ex

```diff
@@ -62,6 +62,35 @@ defmodule Phoenix.Endpoint do
       YourAppWeb.Endpoint.config(:port)
       YourAppWeb.Endpoint.config(:some_config, :default_value)
 
+  ### Module-level configuration
```

Comment:
I don't think we should introduce a new type of configuration. Instead, we could use `debug_errors` to also accept an atom (or a tuple), besides booleans, and act accordingly. WDYT?

Reply:
> I don't think we should introduce a new type of configuration.

Whenever possible, I prefer "module-level" configuration to "application" configuration (when it makes sense for the configuration to be done at compile-time, of course). But that's just a preference. But I guess I could have it either way.

> Instead, we could use debug_errors to also accept an atom (or a tuple), besides booleans, and act accordingly

I don't think we should reuse the `debug_errors` configuration option for this.  IMO, choosing whether to show the debugger when there is an error and which debugger to show are two different concerns. Allowing for the possibility that `debug_errors` can be either a boolean or a tuple also complicates the code with the current architecture. I've actually tried to implement this feature by making `debug_errors` a tuple, and I got into a pretty heavy mess between runtime and compile-time values which I couldn't solve at all (but that may be just my inexperience with compile-time elixir programming).

Reply:
Agreed that we don’t need to reuse the configuration but we should not introduce a new type of configuration just for this, so let’s move that part to the app. :)